optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -33,6 +33,11 @@
     VALID_SET_TYPE = Union[List[lgb.Dataset], Tuple[lgb.Dataset, ...], lgb.Dataset]
 
 
+ELAPSED_SECS_KEY = "lightgbm_tuner:elapsed_secs"
+AVERAGE_ITERATION_TIME_KEY = "lightgbm_tuner:average_iteration_time"
+STEP_NAME_KEY = "lightgbm_tuner:step_name"
+LGBM_PARAMS_KEY = "lightgbm_tuner:lgbm_params"
```

Comment:
I think these can be prefixed with underscores since they aren't public. What do you think?

Reply:
You are right. Users do not use these variables and we can make them private.
Fixed: https://github.com/optuna/optuna/pull/1192/commits/7d606c815b9ac8a2c5c3b468f5aa1c730938aa90